package.json

```diff
@@ -24,6 +24,6 @@
     "inquirer": "^6.2.2",
     "inquirer-autocomplete-prompt": "^1.0.1",
     "netrc": "^0.1.4",
-    "yargs": "^13.2.4"
+    "yargs": "^14.0.0"
   }
```

Comment:
If we do use `chalk` and `stringWidth` the dependencies don't seem to be added to package.json.